build.sbt

```diff
@@ -36,16 +38,26 @@ libraryDependencies ++= Seq(
 
 scalacOptions ++= Seq("-feature")
 
+javaOptions in Universal ++= Seq(
+  s"-Dpidfile.path=/dev/null"
+)
+
```

Comment:
Does this need @philwills recommended options? See something like https://github.com/guardian/membership-frontend/blob/master/frontend/build.sbt#L60-L68